pages/profile/[[...profilePage]].tsx

```diff
@@ -0,0 +1,27 @@
+import { GetServerSideProps } from "next";
```

Comment:
You can simply import the same `profile` component you created for `pages/[network]/[chain]/profile/[[...profilePage]].tsx` (or import this one on the previously mentioned)